engine/security/trust/content_trust.md

```diff
@@ -121,222 +102,318 @@ to create new repositories, it is a good idea to store it offline in hardware.
+can sign an image with the commands [above](#signing-images-with-docker-content-trust)
+, assuming they are stored in a private registry with signing data stored 
+within an attached [Notary Server](/engine/security/trust/deploying_notary/)
+(Such as the Docker Trusted Registry).
 
-### Push trusted content
+Engine Signature Verification prevents the following:
+* `$ docker container run` of an unsigned image.
+* `$ docker build` where the `FROM` image is not signed or is not scratch.
 
-To create signed content for a specific image tag, simply enable DCT
-and push a tagged image. If this is the first time you have pushed an image
-using DCT on your system, the session looks like this:
+DCT does not verify that a running container’s filesystem has not been altered 
+from what was in the image. For example, it does not prevent a container from 
+writing to the filesystem, once the container is running, nor does it prevent 
+the container’s filesystem from being altered on disk. DCT will also not prevent 
+unsigned images from being imported, loaded, or created.
 
-```bash
-$ docker push <username>/trusttest:testing
-The push refers to a repository [docker.io/<username>/trusttest] (len: 1)
-9a61b6b1315e: Image already exists
-902b87aaaec9: Image already exists
-latest: digest: sha256:d02adacee0ac7a5be140adb94fa1dae64f4e71a68696e7f8e7cbf9db8dd49418 size: 3220
-Signing and pushing trust metadata
-You are about to create a new root signing key passphrase. This passphrase
-will be used to protect the most sensitive key in your signing system. Please
-choose a long, complex passphrase and be careful to keep the password and the
-key file itself secure and backed up. It is highly recommended that you use a
-password manager to generate the passphrase and keep it safe. There will be no
-way to recover this key. You can find the key in your config directory.
-Enter passphrase for new root key with id a1d96fb:
-Repeat passphrase for new root key with id a1d96fb:
-Enter passphrase for new repository key with id docker.io/<username>/trusttest (3a932f1):
-Repeat passphrase for new repository key with id docker.io/<username>/trusttest (3a932f1):
-Finished initializing "docker.io/<username>/trusttest"
-```
+### Enabling DCT within the Docker Enterprise Engine 
 
-When you push your first tagged image with DCT enabled, the `docker`
-client recognizes this is your first push and:
+DCT is controlled by the Docker Engine's configuration file. By default this is
+found at `/etc/docker/daemon.json`. For more details on this file can be found 
+[here](/engine/reference/commandline/dockerd/#daemon-configuration-file)
 
- - alerts you that it is creating a new root key
- - requests a passphrase for the root key
- - generates a root key in the `~/.docker/trust` directory
- - requests a passphrase for the repository key
- - generates a repository key in the `~/.docker/trust` directory
+The `content-trust` flag is expecting 2 variables, a `mode` variable instructing
```

Comment:
Technically 3 with "allow-expired-trust-cache", but maybe do away with this sentence entirely since you break down the various variables 1-by-1 below